content/en/_index.html

```diff
@@ -4,17 +4,47 @@
 
 +++
 
-<div style="margin-top: 200px;">
-	<div style="margin: 0 auto; text-align: center;"><h1>Apache Avro™ - a data serialization system.</h1></div>
-	
-	<div class="mx-auto" style="padding-top: 100px; text-align: center; height: 450px;">
-		<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
-			Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
-		</a>
-		<a class="btn btn-lg btn-secondary mr-3 mb-4" href="{{< relref "/project/download" >}}">
-			Download <i class="fas fa-download ml-2 "></i>
-		</a>
+<!-- Copy of default blocks/cover shortcode, but without the td-overlay--dark style class that breaks the white background. -->
+<section id="td-cover-block-0" class="row td-cover-block td-cover-block--height-med td-overlay -bg-white">
+	<div class="container td-overlay__inner">
+		<div class="row">
+			<div class="col-12">
+				<div class="text-center">
+					<h1 class="display-1 mt-0 mt-md-5 pb-4">Your Data — Any Time, Anywhere</h1>
+					<div class="pt-3 lead">
+						<div class="mx-auto">
+							<a class="btn btn-lg btn-primary mr-3 mb-4" href='{{< relref "/docs" >}}'>
+								Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+							</a>
+							<a class="btn btn-lg btn-secondary mr-3 mb-4" href='{{< relref "/project/download" >}}'>
+								Download <i class="fas fa-download ml-2 "></i>
+							</a>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
 	</div>
+</section>
 
-</div>
+{{% blocks/lead color="primary" %}}
+Apache Avro™ is the leading serialization format for record data, and first choice for streaming data pipelines.
+It offers excellent schema evolution, and has implementations for the JVM (Java, Kotlin, Scala, &hellip;), Python, C/C++/C# and soon Rust.
```

Comment:
PHP & Ruby ?

Reply:
Yeah, I neglected those. The Perl implementation, for example, is from 2014.
I've added them now.